ramp-engine/LICENSE

```diff
@@ -1,29 +0,0 @@
-BSD 3-Clause License
```

Comment:
I think we should keep the license in each of the repos?

(in case we package them all separately on pypi (not sure if we plan that), it is nice to have them in the package)

Reply:
Oh ok, yes we can do that.
We probably need to translate the md to an rst then as well